api/v1beta1/disruption_webhook.go

```diff
@@ -626,43 +644,45 @@ func safetyNetCountNotTooLarge(r *Disruption) (bool, string, error) {
 	return false, "", nil
 }
 
-// safetyNetNeitherHostNorPort is the safety net regarding missing host and port values.
-// it will check against all defined hosts in the network disruption spec to see if any of them have a host and a
-// port missing. The more generic a hosts tuple is (Omitting fields such as port), the bigger the blast radius.
-func safetyNetNeitherHostNorPort(r Disruption) bool {
+// safetyNetMissingNetworkFilters is the safety net regarding missing host, service, or port filters.
+// it will check if any filters have been placed to limit which network traffic is disrupted
+func safetyNetMissingNetworkFilters(r Disruption) bool {
 	if r.Spec.Unsafemode != nil && r.Spec.Unsafemode.DisableNeitherHostNorPort {
 		return false
 	}
 
-	// if hosts are not defined, this also falls into the safety net
-	if r.Spec.Network.Hosts == nil || len(r.Spec.Network.Hosts) == 0 {
-		return true
+	if r.Spec.Network == nil {
```

Comment:
This part is not covered by unit test

Reply:
Disagree? We don't test it explicitly, but any test that validates a disruption w/o a network spec and gets no errors, is confirming this works.

Reply:
So does this conditions is required? It looks this condition is made before the call of this one and it's a private method.

Reply:
We do check it in the caller, that's true, but it's still safe to double-check here too? We don't want to rely on the caller and accidentally panic when we check Network.Services